filebeat/input/filestream/input_integration_test.go

```diff
@@ -45,17 +45,17 @@ func TestFilestreamCloseRenamed(t *testing.T) {
 	testlogName := "test.log"
 	inp := env.mustCreateInput(map[string]interface{}{
 		"paths":                                []string{env.abspath(testlogName) + "*"},
-		"prospector.scanner.check_interval":    "1ms",
+		"prospector.scanner.check_interval":    "10ms",
```

Comment:
How does this change fix the test?

Reply:
The problem was that sometimes the prospector detected the rename earlier than the harvester and cancelled the reading before the event could have been sent. By increasing the interval, it is guaranteed that the harvester detects the rename.

Reply:
👍

Reply:
can you put this into a description?